demo/src/app/components/+buttons/demos/checkbox/checkbox.html

```diff
@@ -2,7 +2,9 @@
 
 <div class="btn-group">
   <label class="btn btn-primary" [(ngModel)]="checkModel.left"
-         btnCheckbox>Left</label>
-  <label class="btn btn-primary" [(ngModel)]="checkModel.middle" btnCheckbox>Middle</label>
-  <label class="btn btn-primary" [(ngModel)]="checkModel.right" btnCheckbox>Right</label>
+         btnCheckbox tabindex="0" role="button">Left</label>
```

Comment:
Nothing happens when I focus on button and press `Enter` or `Space`. This is about all buttons with role and tabindex, not just this one